src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/Mocks/PepAuthorizationMockSI.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Xacml;
+using Altinn.Authorization.ABAC.Xacml.JsonProfile;
+using Altinn.Common.PEP.Interfaces;
+
+namespace Altinn.Platform.Storage.IntegrationTest.Mocks.Authentication
+{
+    public class PepAuthorizationMockSI : IPDP
```

Comment:
Could we considert renaming this?? It is my opinion that the only thing mocked in this class is PDP, and that PEP is not involved at all.. The Enforcement point is implemented in AuthorizeInstancesHelper

Reply:
renamed it to PDPMock